addons/point_of_sale/static/src/app/error_handlers/error_handlers.js

```diff
@@ -30,24 +30,17 @@ function rpcErrorHandler(env, error, originalError) {
 }
 registry.category("error_handlers").add("rpcErrorHandler", rpcErrorHandler);
 
-// TODO: consider only showing a notification instead of an error popup in flows that can work offline
-export const urlToMessage = {
-    "/web/dataset/call_kw/pos.order/create_from_ui": _lt(
-        "The order couldn't be sent to the server because you are offline"
-    ),
-};
 function offlineErrorHandler(env, error, originalError) {
     error = identifyError(originalError);
     if (error instanceof ConnectionLostError) {
-        const body =
-            urlToMessage[error.url] ||
-            _t(
-                "The operation couldn't be completed because you are offline. Check your internet connection and try again."
-            );
         env.services.popup.add(OfflineErrorPopup, {
-            title: _t("Couldn't connect to the server"),
-            body,
+            title: _t("You're offline"),
+            body: _t(
+                "Meanwhile connection is back, Odoo Point of Sale will operate limited operations. Check your connection or continue with limited functionalities"
+            ),
+            confirmText: _t("Continue with limited functionalities"),
```

Comment:
It is already the default values of the OfflineErrorPopup, that is strange to put them again here but I'm ok with it.